src/Recommendations.js

```diff
@@ -51,12 +53,22 @@ const buildSearchParamsFromRecommendations_TEMPORARY_BETA = (record, props) => {
       hitsPerPage = record.recommendations.length;
     }
   }
-
+  if (this.model === 'bought-together') {
+    return {
+      optionalFilters: [],
+      filters:
+        (recoFilters.length > 0 &&
+          composeFilters(recoFilters)) ||
+        null,
+      facetFilters: (recoFilters.length > 0 && props.facetFilters) || [],
```

Comment:
I don't get why you would only do the filtering if there are some recommentations.